.nvmrc

```diff
@@ -0,0 +1 @@
+6.10.0
```

Comment:
I think it is reasonable to provide a default version of Node.js for the `elm` builds that is different from `node_js` builds.

Do take care to provide useful information if the user ends up specifying something that doesn't work.